tests/features/piwik.feature

```diff
@@ -27,7 +27,10 @@ Feature: Check Piwik
     And the response should contain the meta tag with the "X-UA-Compatible" name and the "IE=edge" content
 
   Scenario: Check if the PIWIK script flags forbidden pages
-    Given I go to "ecaslogout"
+    Given these modules are enabled
```

Comment:
The module is enabled by default in a fresh install of the platform. Why do we need to explicitly enable it here while we do not do it elsewhere like in `ecas.feature` scenario.

I raise the point because this step force the site behaviour compared to the default platform one. We change then the test parametrising where it is not requested.

Reply:
It is a dependency in profiles/multisite_drupal_standard/multisite_drupal_standard.info and so it's not enabled for communities profile. Hence we need to ensure it's enabled here.

Reply:
I completely forgot communities! 😄 
Thanks!

👍